python/step_4.html

```diff
@@ -0,0 +1,73 @@
+<!--TODO this is way too long, but I don't know how to break it down, there is some theory on Rest and then some necessary info on pokerwars-->
```

Comment:
IMO there's a natural break at "theory in action"

Reply:
cool, yes same for me, I didn't break it before because I was not sure to have 2 steps in a row without interactions